docs/pilots/0.2.0-rc.1-pilot-selection.md

```
@@ -0,0 +1,113 @@
+   - Insert an unreceipted syntax finding (e.g. `unwrap()` in non-test code).
+   - Verify `cargo-allow check --mode no-new` exits non-zero and points to `why --plan`.
+   - Remove negative control and verify return to clean green status.
+4. **Friction Logging**: Post any unexpected CLI friction or diagnostic ambiguities to [#3151](https://github.com/EffortlessMetrics/cargo-allow/issues/3151).
+
+### Brownfield Adoption Packet (`xchecker` -> #2467)
+1. **Installation & Discovery**:
+   ```bash
+   cargo install cargo-allow --version 0.2.0-rc.1 --locked
+   cargo-allow audit --format json --output target/cargo-allow/brownfield-audit.json
+   cargo-allow worklist
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)